.claude/commands/debug.md

```
@@ -48,6 +72,81 @@ Protocol Phases:
+
+Session Folder: temp/DEBUG_REPORTS/session-20260206-001/
+
+Phase: SETUP -> Use '/debug step 1-reproduce "findings"' to begin
+       All agents investigate in parallel during phases 1-3.
+
+Team Commands:
+  /debug assign <agent> <zone>    - Reassign agent to zone
+  /debug findings <agent>         - View agent's findings
+  /debug merge                    - Trigger merge resolution
+  /debug conflicts                - Show detected conflicts
+  /debug status --team            - Show all agents' status
```

markdownlint-cli2 (0.20.0)

[warning] Fenced code blocks should be surrounded by blank lines

(MD031, blanks-around-fences)